stable/drone/templates/deployment-agent.yaml

```diff
@@ -45,7 +49,7 @@ spec:
           protocol: TCP
         env:
           - name: DRONE_RPC_SERVER
-            value: {{ template "drone.fullname" . }}-grpc:9000
+            value: http://{{ template "drone.fullname" . }}
```

Comment:
what about?
[the grpc service](https://github.com/helm/charts/blob/b2425d3e1ea4e446086bdd7481e868a2ba584c29/stable/drone/templates/service-grpc.yaml)